debian/python-eos-data-distribution.install

```diff
@@ -0,0 +1,3 @@
+eos_data_distribution/producers/70-edd-usb-producer.rules usr/lib/udev/rules.d
+eos_data_distribution/producers/edd-usb-producer.service lib/systemd/system
```

Comment:
As discussed on the call today, it’s best to do that later (if at all), if the existing monolithic package starts pulling in too many dependencies across the system; i.e. only if there’s a good reason to want to install part of eos-data-distribution without any of the rest of it. At the moment, I think it’s reasonable to install it all together.